app/app.js

```diff
@@ -5,11 +5,15 @@ import Relay from 'react-relay';
 import {
   Router,
   Reducer,
-  Scene
+  Scene,
+  Actions
 } from 'react-native-router-flux';
 import RelayRenderer from './shared/relayComponentRenderer';
 import MyLibrary from './components/myLibrary/myLibrary';
 
+import imgArrowDown from './resources/arrow_down.png';
```

Comment:
Is this the standard way? I
I was going to use this way 
const imgArrowDown = require('');
I don't know which one is the right.